public class GetException {
    public static void main(String[] args) {
        int number = 0;
        System.out.println(5/number);

        System.out.println("This will not be printed because the program already crashed");
    }
}
